constraints.txt

```diff
@@ -34,13 +34,13 @@ black==21.6b0
     # via dallinger
 bleach==3.3.0
     # via nbconvert
-boto3==1.17.105
+boto3==1.17.93
```

Comment:
I'm curious why we're downgrading so many dependencies on this branch.

Reply:
Good catch, thanks! Tihs was my mistake while rebasing.